src/libraries/Common/src/Interop/Windows/SspiCli/SafeDeleteContext.cs

```diff
@@ -27,7 +27,7 @@ internal abstract partial class SafeDeleteContext : SafeHandle
         //
         internal Interop.SspiCli.CredHandle _handle;
 
-        protected SafeDeleteContext() : base(IntPtr.Zero, true)
+        public SafeDeleteContext() : base(IntPtr.Zero, true)
```

Comment:
This class is `abstract`

Reply:
Repeated elsewhere. Please revert all `abstract` classes so they don't have `public` constructors.